src/test/scala/com/github/music/of/the/ainur/almaren/Test.scala

```diff
@@ -60,12 +60,14 @@ class Test extends FunSuite with BeforeAndAfter {
 
   val moviesDf = spark.table(testTable)
 
-  test(testSourceTargetJdbc(moviesDf), moviesDf, "SourceTargetJdbcTest")
+ // test(testSourceTargetJdbc(moviesDf), moviesDf, "SourceTargetJdbcTest")
   test(testSourceTargetJdbcUserPassword(moviesDf), moviesDf, "SourceTargetJdbcTestUserPassword")
   test(testSourceFile("parquet","src/test/resources/sample_data/emp.parquet"),
     spark.read.parquet("src/test/resources/sample_output/employee.parquet"),"SourceParquetFileTest")
   test(testSourceFile("avro","src/test/resources/sample_data/emp.avro"),
     spark.read.parquet("src/test/resources/sample_output/employee.parquet"),"SourceAvroFileTest")
+  test(testTargetFile("parquet","src/test/resources/sample_target/target.parquet",SaveMode.Overwrite,Map(),List("first_name"),(4,List("last_name")),List("first_name")),df,"TargetParquetFileTest")
```

Comment:
The test should see if the data was proper partitioned/bucket/sort etc. You can do this listing the partitions/buckets and for sort is easy.